content/install_maintain/installation/installing-cluster.md

```diff
@@ -248,96 +280,160 @@ whereas Cloudify best-practice is three nodes.
 
-For the first RabbitMQ, configure the following settings in `/etc/cloudify/config.yaml`:
+For the first RabbitMQ node, configure the following settings in `/etc/cloudify/config.yaml`:
 
 ```yaml
+manager:
+  private_ip: <ip of this host>
+  public_ip: <ip of this host>
+
 rabbitmq:
 
-    username: '<secure username for queue management>'
-    password: '<secure password for queue management>'
-
-    cluster_members:
-        <short host name of rabbit server 1- e.g. using `hostname -s`>:
-            networks:
-                default: <private ip of rabbit server 1>
-                <other network name>: <address for this node on `other network`>
-        <short host name of rabbit server 2>:
-            networks:
-                default: <private ip of rabbit server 2>
-                <other network name>: <address for this node on `other network`>
-        <short host name of rabbit server 3>:
-            networks:
-                default: <private ip of rabbit server 3>
-                <other network name>: <address for this node on `other network`>
+  username: '<secure username for queue management>'
+  password: '<secure password for queue management>'
+    
+  cluster_members:
+    <host name of rabbit server 1>:
+      networks:
+        default: <private ip of rabbit server 1>
+    <host name of rabbit server 2>:
+        networks:
+            default: <private ip of rabbit server 2>
+    <host name of rabbit server 3>:
+        networks:
+            default: <private ip of rabbit server 3>
+    
+  cert_path: '<path to certificate for this server>'
+  key_path: '<path to key for this server>'
+  ca_path: '<path to ca certificate>'
+    
+  nodename: '<short host name of this rabbit server>'
+    
+  # Should be the same on all nodes
+  erlang_cookie: '<a strong secret string (password-like)>'
+
+# For monitoring service(status reporter)
+prometheus:
+  credentials:
+    username: <monitoring username>
+    password: <strong password for monitoring user>
+
+  cert_path: <certificate for prometheus, cert_path of this rabbitmq can be used>
+  key_path: <key for promethus, key_path of rabbitmq can be used>
+  ca_path: <ca for promethus, key_path of rabbitmq can be used>'
+services_to_install:
+  - queue_service
+  - monitoring_service
+```
 
-    cert_path: '<path to certificate for this server>'
-    key_path: '<path to key for this server>'
-    ca_path: '<path to ca certificate>'
+For the rest of rabbitmq nodes, just add join_cluster in the rabbitmq section, as in the following config.yaml:
+ 
+```yaml
+manager:
+  private_ip: <ip of this host>
+  public_ip: <ip of this host>
 
-    nodename: '<short host name of this rabbit server>'
-    
-    # Should be the same on all nodes
-    erlang_cookie: '<a strong secret string (password-like)>'
+rabbitmq:
 
+  username: '<secure username for queue management>'
+  password: '<secure password for queue management>'
+    
+  cluster_members:
+    <host name of rabbit server 1>:
```

Comment:
Each of these should probably be specified as <short host name of rabbit server #>

Reply:
(because that's how nodename is specified too, so otherwise it looks like they can be different)

Reply:
maybe we can just write "node name of first rabbit server"